core/templates/dev/head/pages/exploration-player-page/exploration-player-page.controller.ts

```diff
@@ -105,6 +105,8 @@ require('domain/exploration/ReadOnlyExplorationBackendApiService.ts');
 
 require('objects/objectComponentsRequiresForPlayers.ts');
 
+require('services/AutoplayedVideosService.ts');
+require('services/CodeNormalizerService.ts');
```

Comment:
Please move this directly after line 82.

Reply:
Done!